### PR TITLE
SIP minutes: 2019 November

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+* part 2 (06:43): <https://www.youtube.com/watch?v=jFjjYybY_wY>
+
+Minutes were taken by Dale Wijnand.
+
+## Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Darja Jovanovic ([@darjutak](https://github.com/darjutak)), Process Lead
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
+* Guillaume Marters ([@smarter](https://github.com/smarter), EPFL
```
Comment (SethTisue) [inline _sips/minutes/2019-11-27-sip-minutes.md]: s/Marters/Martres/

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+* What happens to the current SIPs?
+* What is the timeline for them?
+* Who can make new proposals?
+* Which changes to the process were made in November 2018?
+* And which changes should be made going forward?
+
+Darja presented the topic.  The background is that in November 2018 the SIP Committee accepted the "Dotty team
+proposal of changes", which wasn't like the regular SIP proposals but is more like a large number of proposals
+for Scala 3.  Those individual proposal still need to be individually reviewed, but doing so will take time.
```
Comment (SethTisue) [inline _sips/minutes/2019-11-27-sip-minutes.md]: s/proposal/proposals/

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+1. Make a commitment to update the state of all SIPs by March 2020;
+2. Change the process to introduce the concept of a "SIP Champion", which is a member of the SIP Committee that
+   a contributor finds to progress a Pre-SIP idea into a SIP and to champion the SIP through the process
+
+### Dotty feature freeze is coming soon - what does that mean?
+
+Guillaume presented.
+
+A month ago a thread was created on the Contributors forum announcing that the next Dotty release will
+initialise a feature freeze.  What that means is that new things won't be added, instead existing changes will
```
Comment (SethTisue) [inline _sips/minutes/2019-11-27-sip-minutes.md]: s/initialise/initiate/

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+be refined.  That doesn't mean that new SIPs can't be proposed, but it does mean that such SIPs should adjust
+their expectations that its very unlikely they'll be able to target Scala 3.0.
```
Comment (SethTisue) [inline _sips/minutes/2019-11-27-sip-minutes.md]: maybe add "and would have to wait for some future 3.x release"

**Review by SethTisue — COMMENTED** [review]
LGTM, I made only very minor comments

and, one suggestion: could this say something about when the next meeting/meetings/retreats are expected to occur?

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+1. Make a commitment to update the state of all SIPs by March 2020;
+2. Change the process to introduce the concept of a "SIP Champion", which is a member of the SIP Committee that
+   a contributor finds to progress a Pre-SIP idea into a SIP and to champion the SIP through the process
+
+### Dotty feature freeze is coming soon - what does that mean?
+
+Guillaume presented.
+
+A month ago a thread was created on the Contributors forum announcing that the next Dotty release will
```
Comment (darjutak) [inline _sips/minutes/2019-11-27-sip-minutes.md]:
```suggestion
A month ago a thread was created on the [Contributors forum announcing that the next Dotty release](https://contributors.scala-lang.org/t/preparing-for-feature-freeze) will
```

File: _sips/minutes/2019-11-27-sip-minutes.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+Guillaume gave a quick update: a new thread was created on the Contributors forum to discuss how Scala's
+implicits are being revised in Scala 3: <https://contributors.scala-lang.org/t/updated-proposal-revisiting-implicits/3821>
+
+Because the discussion has returned to debating the naming change, he intends to split the thread so that
+also-important discussions around the semantics changes aren't drowned out.
+
+## Next
+
+The next meeting will be in December, but also the Committee intends to have another retreat in March 2020.
```
Comment (darjutak) [inline _sips/minutes/2019-11-27-sip-minutes.md]:
```suggestion
The next meeting will be December 18th at 5 PM CET, but also the Committee intends to have another retreat in March 2020.
```

**Review by darjutak — CHANGES_REQUESTED** [review]
Looks excellent to me, thank you! (just a couple of minor additions)

**Review by sjrd — APPROVED** [review]

**Comment by dwijnand** [comment]
Thanks, I'll PR it now.

**Pull request closed without merging**